index.html

```diff
@@ -12,9 +12,9 @@
     <img src="https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.collinsdictionary.com%2Fdictionary%2Fenglish%2Flake&psig=AOvVaw1e6qxrewvh4DFgrQ24h_Wi&ust=1639075797144000&source=images&cd=vfe&ved=0CAsQjRxqFwoTCPCDqKfv1PQCFQAAAAAdAAAAABAD" alt="" id="header-img">
     <nav id="nav-bar">
 
-        <button class="nav-link"> First button</button>
-        <button class="nav-link"> Second button</button>
-        <button class="nav-link"> Third button</button>
+        <button class="nav-link">First button</button>
+        <button class="nav-link">Second button</button>
+        <button class="nav-link">Third button</button>
         <button>Fourth button</button>
```

Comment:
Adding type attributes to Html tags would be more semantic. Kindly make the changes.